src/studio/src/designer/frontend/shared/components/AltinnInputField.tsx

```diff
@@ -146,8 +149,23 @@ class AltinnInputFieldComponent extends React.Component<IAltinnInputFieldCompone
             multiline={!!this.props.textAreaRows}
             minRows={this.props.textAreaRows || null}
             InputProps={{
-              disableUnderline: true,
-              classes: { root: classNames(classes.inputFieldText) },
+              classes: {
+                root: classes.inputFieldText,
+              },
+            }}
+            inputProps={{
```

Comment:
Two different cased props with the same name, is that right?

Reply:
Yes.

https://mui.com/material-ui/api/text-field/

![image](https://user-images.githubusercontent.com/589907/197495723-a3a0fb2d-05d3-41f9-b208-7ad61e5db6e0.png)